docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.2'
 
 services:
   db:
-    image: postgres
+    image: postgres:12
```

Comment:
@danparisd I think it should be 11. At least we have 11 on the box.

Reply:
12 works and gives us another year of support. It will break the NUC if we git pull, but we can fix it, or start over on that one.